### PR TITLE
Added blog on contributor workflow

### PR DESCRIPTION
**Pull request by tvvignesh**
This PR adds the blog on contributor workflow as suggested by @Urigo

### CODE REVIEW

File: pages/blog/oss-contributor-workflow.mdx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+But does creating a guide suffice to achieve our goal? As you guessed, definitely not, and the guide was just a start to an amazing journey ahead.
+
+The next thing we did was to prepare a set of [standard issue templates](https://github.com/Urigo/graphql-mesh/tree/master/.github/ISSUE_TEMPLATE) which remains almost the same across all our repositories. Again, a work in progress we will tweak this as the need arises. This gives us an ability to suggest a standard template to file issues and PRs to all the new contributors asking for all the important details like the environment and version they use, possible reproductions/screenshots, additional context and also providing them with checklists to be actioned upon when filing an issue.
+
+While all this might seem very simple, doing this has a long term benefit of communicating your expectations as a maintainer to the community when they are looking to contribute. Now, this is something which some of the projects tend to ignore.
+
+Then we started thinking about the next important thing which we can address to make both maintainer's and contributor's lives easier - LABELS
```
Comment (Urigo) [inline pages/blog/oss-contributor-workflow.mdx:54]: Maybe talk about labels before issue templates?
It's a bigger change

Comment (Urigo) [reply pages/blog/oss-contributor-workflow.mdx:54]: with header that's called something like "Process based Github Labels" or something?

File: pages/blog/oss-contributor-workflow.mdx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+Doing this was pretty simple. You can find the repo [here](https://github.com/the-guild-org/shared-resources/) which is used as the master repository to push all the labels downstream and we still retain the repo specific labels in their own repository ([like this](https://github.com/Urigo/graphql-mesh/blob/master/.github/labels.yml))
+
+Now that a technical proof of concept was complete, the next question we had was "What labels should we standardize on as the master labels?"
+
+Now, [Urigo](https://twitter.com/UriGoldshtein) had an interesting thought process - since we had already published the contributor workflow guide, why not we label issues keeping the same in mind? All we needed to do was push an issue towards progress from Step 1 to Step N
+
```
Comment (Urigo) [inline pages/blog/oss-contributor-workflow.mdx:69]: "The goal here was to make it more obvious for anyone who arrives to an issue where exactly it stands and most importantly, what can you do to help advance the solution"

File: pages/blog/oss-contributor-workflow.mdx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+Doing this was pretty simple. You can find the repo [here](https://github.com/the-guild-org/shared-resources/) which is used as the master repository to push all the labels downstream and we still retain the repo specific labels in their own repository ([like this](https://github.com/Urigo/graphql-mesh/blob/master/.github/labels.yml))
+
+Now that a technical proof of concept was complete, the next question we had was "What labels should we standardize on as the master labels?"
+
+Now, [Urigo](https://twitter.com/UriGoldshtein) had an interesting thought process - since we had already published the contributor workflow guide, why not we label issues keeping the same in mind? All we needed to do was push an issue towards progress from Step 1 to Step N
```
Comment (Urigo) [inline pages/blog/oss-contributor-workflow.mdx:68]: link to the workflow

File: pages/blog/oss-contributor-workflow.mdx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+- `stage/0-issue-prerequisites`
+- `stage/1-reproduction`
+- `stage/2-failing-test`
+- `stage/3-local-solution`
+- `stage/4-pull-request`
+- `stage/5-alpha-release-testing`
+- `stage/6-released`
+
+which is how almost workflows look like for every project (Guild or even otherwise) - This is again the initial iteration and we will have more room for improvement over time and you may see more/lesser stages over time.
```
Comment (Urigo) [inline pages/blog/oss-contributor-workflow.mdx:80]: most/almost?

**Review by Urigo — CHANGES_REQUESTED** [review]
Great post!

I've added some small notes.

I think the most important thing for me to emphasize is that what we really care about and trying to solve is making it easier for people who never contributed to open source to start participating, opening that world to them.

The initial step, the first time is the hardest.

We want to show them the path, what's possible and how to get there.

If you can add a bit more about these types of things it would be awesome!

**Comment by kamilkisiela** [comment]
Preview: https://the-guild-webs-git-fork-tvvignesh-blog-contrib-workflow-15224a.vercel.app/blog/oss-contributor-workflow

File: pages/blog/oss-contributor-workflow.mdx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  thumbnail: '/blog-assets/oss-contributor-workflow/cover.png',
+};
+
+export default withArticle({ ...meta });
+
+## Orchestrating the Open Source Contribution Workflow
+
+Open source has revolutionized the way we build, collaborate and release software and there are definitely a lot of instances where the community of contributors and maintainers have taken the lead to make the projects thrive in the ecosystem and this is really exciting to see.
+
+But as anything goes, it is not without its own set of challenges which advocates us to think of interesting solutions. This stands true in our case as well.
```
[the remark this answers is not in the document]
Comment (tvvignesh) [reply pages/blog/oss-contributor-workflow.mdx:19]: Not sure why we are talking about the "career" aspect here - is it really relevant?

File: pages/blog/oss-contributor-workflow.mdx (hunk 4 above)
Comment (tvvignesh) [reply pages/blog/oss-contributor-workflow.mdx:80]: Thanks for catching that. It should be either `almost all` or `most`

**Comment by tvvignesh** [comment]
@Urigo Addressed all your feedback except [this](https://github.com/the-guild-org/the-guild-website/pull/346#discussion_r616817413) which I was not sure of.

File: pages/blog/oss-contributor-workflow.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (Urigo) [reply pages/blog/oss-contributor-workflow.mdx]: yes maybe not that. I guess my point is around that there are benefits for people to contribute to open source and if its hard it's a shame.... I'm sure you can write something much better than I did :)

File: pages/blog/oss-contributor-workflow.mdx [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+### TLD;R
+
+- We released [The Guild's contributor guide](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md)
+- [Contributor workflow](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#a-typical-contributor-workflow)
+- [Video tutorials on how to contribute](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#contributor-videos)
+- [The Guild's tooling](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#the-tooling) for maintaining open source libraries
+- A [new system](https://github.com/the-guild-org/shared-resources/) for managing our repositories (Process based labeling system, issue templates)
+
+Open source has revolutionized the way we build, collaborate and release software and there are definitely a lot of instances where the community of contributors and maintainers have taken the lead to make the projects thrive in the ecosystem and this is really exciting to see.
+Making open source more approachable has beneficial for the maintainers, users and the contributors themselves - opening new paths for them that seemed harder before.
```
Comment (Urigo) [inline pages/blog/oss-contributor-workflow.mdx:26]: is beneficial?  has benefits?
It's worth trying opening new paths?
something like that?

File: pages/blog/oss-contributor-workflow.mdx [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+## Orchestrating the Open Source Contribution Workflow
+
+But as anything goes, it is not without its own set of challenges. The number of open source contributors is a tiny fracture compared to the number of users of our libraries.
+
+This advocates us to think of interesting solutions, as we believe many developers are missing out on these opportunities to advance themselves and their careers.
+
```
Comment (Urigo) [inline pages/blog/oss-contributor-workflow.mdx:33]: "We see that improving that is [The Guild's core mission!](https://the-guild.dev/about-us)"

**Review by Urigo — APPROVED** [review]
as usual, you take my wording and make it 10x better :)

Added small notes but at this point it is ready.

We'll publish it tomorrow

**Pull request merged**